Makefile

```diff
@@ -10,6 +10,8 @@ FLUENTBIT_VERSION = 1.9.10
 FLUENTBIT_IMAGE ?= ${RP_IMAGE_ACR}.azurecr.io/fluentbit:$(FLUENTBIT_VERSION)-cm$(MARINER_VERSION)
 AUTOREST_VERSION = 3.6.3
 AUTOREST_IMAGE = quay.io/openshift-on-azure/autorest:${AUTOREST_VERSION}
+GUARDRAILS_VERSION = 3.10.0
+GUARDRAILS_IMAGE ?= ${RP_IMAGE_ACR}.azurecr.io/guardrails:$(GUARDRAILS_VERSION)
```

Comment:
nit: probably GATEKEEPER_ is a bit better as it is the version and image of gatekeeper

Reply:
Made the change.